src/test/common/test_type_check_decorator.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2021 Research Institute of Systems Planning, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from typing import Dict, List, Union
+
+from caret_analyze.common.type_check_decorator import type_check_decorator
+from caret_analyze.exceptions import UnsupportedTypeError
+
+import pytest
+
+
+@pytest.mark.skipif('GITHUB_ACTION' in os.environ, reason='requires pydantic>=1.5')
```

Comment:
This line is  not required because pytest on github actions are done currently in humble.